### PR TITLE
WD-9167 Rebrand server power 

### PR DESCRIPTION
**Pull request by carkod**
## QA

- Compare https://ubuntu-com-13678.demos.haus/download/server/power against [copy docs](https://docs.google.com/document/d/1e-vav_b54KlKsi92w3rDu9J14O0gpp8yAanEAIImlE8/edit)
- Compare against design screenshot below
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-9164

## Screenshots

[
power
](url)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/download/server/power.html [hunk 1]
```diff
@@ -5,44 +5,119 @@
 {% block meta_copydoc %}https://docs.google.com/document/d/1e-vav_b54KlKsi92w3rDu9J14O0gpp8yAanEAIImlE8/edit{% endblock meta_copydoc %}
 
 {% block content %}
-<section class="p-strip--suru-topped is-bordered">
-  <div class="row">
-    <div class="col-10">
+<section class="p-strip is-shallow">
```
Comment (lyubomir-popov) [inline templates/download/server/power.html:8]:
```suggestion
<section class="p-strip is-shallow u-no-padding--bottom">
```

File: templates/download/server/power.html [hunk 2]
```diff
@@ -5,44 +5,119 @@
...
+  <div class="row--50-50">
```
Comment (lyubomir-popov) [inline templates/download/server/power.html:9]:
```suggestion
  <div class="row--50-50 p-section">
```

**Review by lyubomir-popov — COMMENTED** [review]
some suggestions

**Comment by lyubomir-popov** [comment]
image
Can we indent h5s one column on medium screens? `col-start-medium-2 col-medium-2` should do it

File: templates/download/server/power.html [hunk 3]
```diff
@@ -5,44 +5,103 @@
...
   </div>
+  <div class="row">
+    <div class="col-9 col-start-large-4 col-start-medium-2 col-medium-5">
+      <hr class="p-rule" />
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h2 class="p-heading--5">Ubuntu Server</h2>
+        </div>
+        <div class="col-6 col-medium-3">
+          <hr class="p-rule" />
```
Comment (juanruitina) [inline templates/download/server/power.html:27]: This rule seems unnecessary, please remove it

File: templates/download/server/power.html [hunk 4]
```diff
@@ -5,44 +5,103 @@
...
+          <p>This is the iso image of the Ubuntu Server installer.</p>
```
Comment (juanruitina) [inline templates/download/server/power.html:28]: Please capitalise "ISO" here

**Comment by juanruitina** [comment]
The spacing above and below the "Ubuntu Server" section doesn't match the mockup, and I think that affects the sense of hierarchy.

Little change: please adjust the two CTA as per the copy (drop "Ubuntu" from them). I also left a couple minor comments in the code.

**Comment by juanruitina** [comment]
Spacing still seems a bit excessive on top of the Ubuntu Server section, but there's better hierarchy now and it's a visual matter anyway, so UX+1.

**Comment by carkod** [comment]
> The spacing above and below the "Ubuntu Server" section doesn't match the mockup, and I think that affects the sense of hierarchy, please fix.
> 
> Little change: please adjust the two CTA as per the copy (drop "Ubuntu" from them). I also left a couple minor comments in the code.

That spacing was suggested by @lyubomir-popov https://github.com/canonical/ubuntu.com/pull/13678#discussion_r1530412075

File: templates/download/server/power.html [hunk 5]
```diff
@@ -3,46 +3,104 @@
...
+{% block body_class %}is-paper{% endblock body_class %}
 
 {% block content %}
-<section class="p-strip--suru-topped is-bordered">
-  <div class="row">
-    <div class="col-10">
+<section class="p-strip is-shallow u-no-padding--bottom">
+  <div class="row--50-50 p-section--shallow">
+    <div class="col">
       <h1>Ubuntu Server for IBM POWER</h1>
```
Comment (lizzochek) [inline templates/download/server/power.html:12]: This heading is in 2 lines in design, so maybe add a br

**Review by lizzochek — APPROVED** [review]

**Pull request merged**